<ui version="4.0" >
 <class>numpad</class>
 <widget class="QDialog" name="numpad" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>NumPad</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>120</y>
     <width>381</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QPushButton" name="pBtn1" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn2" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>10</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn3" >
    <property name="geometry" >
     <rect>
      <x>250</x>
      <y>10</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn6" >
    <property name="geometry" >
     <rect>
      <x>250</x>
      <y>80</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn5" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>80</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn4" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn9" >
    <property name="geometry" >
     <rect>
      <x>250</x>
      <y>150</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn7" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>150</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn8" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>150</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtnClr" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>220</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtnd" >
    <property name="geometry" >
     <rect>
      <x>250</x>
      <y>220</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn0" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>220</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pBtnEnt" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>440</y>
     <width>361</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QLabel" name="dispBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground" >
    <bool>true</bool>
   </property>
   <property name="frameShape" >
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="msgLabel" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>13</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
